assets/scripts/aircraft.js

```diff
@@ -1805,14 +1805,32 @@ var Aircraft=Fiber.extend(function() {
         // ALTITUDE
 
         var distance = null;
-        var expedite_factor = 1.7;
+        var expedite_factor = 1.5;
         this.trend = 0;
         if(this.target.altitude < this.altitude - 0.02) {
-          distance = -this.model.rate.descent * game_delta() / expedite_factor;
+          distance = -this.model.rate.descent/60 * game_delta();
           if(this.mode == "landing") distance *= 3;
           this.trend -= 1;
         } else if(this.target.altitude > this.altitude + 0.02) {
-          distance =  this.model.rate.ascent  * game_delta() / expedite_factor;
+          function getClimbRate(ac) {
+            var a = ac.altitude;
+            var r = ac.model.rate.climb;
+            var c = ac.model.ceiling;
+            if(ac.model.engines.type == "J") var serviceCeilingClimbRate = 500;
+            else var serviceCeilingClimbRate = 100;
+            if(ac.altitude < 36152) { // in troposphere
+              var cr_uncorr = r*420.7* ((1.232*Math.pow((518.6 - 0.00356*a)/518.6, 5.256)) / (518.6 - 0.00356*a));
+              var cr_current = cr_uncorr - (a/c*cr_uncorr) + (a/c*serviceCeilingClimbRate);
+            }
+            else { // in lower stratosphere
```

Comment:
Set the else to return the uncorrected base climb rate (obviously wrong), and marked it clearly as a placeholder in 65a5530.